src/content/docs/iast/iast-configuration.mdx

```diff
@@ -0,0 +1,380 @@
+---
+title: IAST configuration
+tags:
+    - IAST
+    - New Relic IAST
+    - Configuration
+metaDescription: Find out how to configure IAST.
+freshnessValidatedDate: 2024-09-16
+---
+
+You can configure your IAST to handle scan scheduling. These configurations allow you to exclude certain APIs, parameters, and vulnerability categories from IAST analysis. You can also delay IAST scans or schedule them for specific times of the day.
+
+## Scheduling IAST scans [#scheduling-iast-scans]
+
+You can start and stop your scheduled IAST scans using 2 variables. These variables allow you to set specific times for the IAST scan or add a delay to the IAST scan start time from the application.
```

Comment:
Should this be '2' or 'two'?

Reply:
I think it's 2 according to our style guide ;-): [Numbers, dates, and times](https://docs.newrelic.com/docs/style-guide/ui-writing/numbers-dates-and-times/#basics)